docs/advanced_topics/api/v2/configuration.rst

```diff
@@ -238,7 +238,8 @@ This would add the following to the JSON:
             "height": 1125
         },
         "feed_image_thumbnail": {
-            "url": "/media/images/a_test_image.fill-100x100.jpg",
+            "url": "http://www.example.com/media/images/a_test_image.fill-100x100.jpg",
+            "full_url": "/media/images/a_test_image.fill-100x100.jpg",
```

Comment:
Should these values be the other way around? The `url` should be the one without https (i.e. current behaviour)

Reply:
Sorry!

Reply:
No need to be sorry, it's why we do reviews. It's super easy to miss things - another set of eyes always helps.